superset-frontend/src/components/URLShortLinkButton/URLShortLinkButton.test.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { render, screen } from 'spec/helpers/testing-library';
+import userEvent from '@testing-library/user-event';
+import fetchMock from 'fetch-mock';
+import URLShortLinkButton from 'src/components/URLShortLinkButton';
+import ToastPresenter from 'src/messageToasts/containers/ToastPresenter';
+
+const fakeUrl = 'http://fakeurl.com';
```

Comment:
I wonder how much traffic that site gets from random things like this.

Reply:
Hahaha so true! At least in our case, we don't actually reach them 😉